LICENSE

```diff
@@ -31,16 +31,16 @@
       "License" shall mean the terms and conditions for use, reproduction,
       and distribution as defined by Sections 1 through 9 of this document.
 
-      "Licensor" shall mean the copyright owner or entity authorized by
+      "Licensor" shall mean the copyright owner or Tag authorized by
```

Comment:
All of the changes in this file need to be reverted. It's the License for the project.